views/js/qtiCommonRenderer/renderers/interactions/EndAttemptInteraction.js

```diff
@@ -0,0 +1,125 @@
+define([
+    'lodash',
+    'tpl!taoQtiItem/qtiCommonRenderer/tpl/interactions/endAttemptInteraction',
+    'taoQtiItem/qtiCommonRenderer/helpers/Helper',
+    'taoQtiItem/qtiCommonRenderer/helpers/PciResponse',
+    'i18n'
+], function(_, tpl, Helper, pciResponse, __){
```

Comment:
Add the `"use strict";` declaration after the main function (same for the other files)